### PR TITLE
docs: add "Administration" section

### PR DESCRIPTION
**Pull request by mattdibi**
Added "Administration" section.

### CODE REVIEW

**Review by marcellorinaldo — CHANGES_REQUESTED** [review]
Please remove the section Remote Management with Everyware Cloud with related images.

**Comment by mattdibi** [comment]
@marcellorinaldo Removed section and fixed merge conflict. Should be good to go now.

File: docs/administration/snapshot-management.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Snapshot Management
+
+The overall configuration of Kura is stored in an XML file called a snapshot. This file includes all of the parameters for every service running in Kura. The original configuration file is named **snapshot_0.xml**. This section describes how snapshots may be used.
+
+Each time a configuration change is made to one of the Kura components, a new XML file is created using the naming convention **snapshot_[time as a long integer].xml**. The nine most recent snapshots are saved, as well as the original snapshot 0.
+
+## How to Access Snapshots
+To display snapshots using the [Gateway Administration Console](console.html), select **Settings** from the **System** area, and then click on the **Snapshots** tab. The following three operations are available: **Download**, **Upload and Apply**, and **Rollback**.
```
Comment (marcellorinaldo) [inline docs/administration/snapshot-management.md]: Link is broker, please use `../gateway-configuration/gateway-administration-console.md`

File: docs/administration/snapshot-management.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+## How to Use Snapshots
+
+### Download
+
+The **Download** option provides the ability to save a snapshot file onto your computer. This file may then be edited, uploaded back to the device, or transferred to another equivalent device.
+
+Starting from Kura 5.1, the snapshot can be downloaded in two formats:
+
+* **XML**: The original XML snapshot format.
+* **JSON**: The JSON format used by the [Configuration v2 REST APIs and CONF-V2 request handler](doc:configuration-v2-rest-apis-and-conf-v2-request-handler). For example the downloaded snapshot can be used as is as a body for the [PUT/configurableComponents/configurations/_update](doc:configuration-v2-rest-apis-and-conf-v2-request-handler#putconfigurablecomponentsconfigurations_update) request. The `takeSnapshot` parameter specified by the CONF-V2 request is missing from the downloaded JSON file, if that parameter is not specified, a new snapshot will be created by default.
```
Comment (marcellorinaldo) [inline docs/administration/snapshot-management.md]: Broken Link, should point to `../core-services/configuration-service-rest-v2.md`, same for the PUT link

File: docs/administration/remote-management-kapua.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+## Snapshots
+
+As described in [Snapshot Management](../snapshot-management), the overall Kura configuration, including the new installed applications, is stored in a snapshot xml file. The Eclipse Kapua Console also provides options to **Download**, **Upload and Apply**, or **Rollback** snapshots as shown in the screen capture below.
+
+![](images/KapuaSnapshots.png){ style="border-radius: 7px;"}
+
+## Remote Command Execution from Eclipse Kapua Web Console
+
+The Eclipse Kapua Console provides the ability to run system commands directly on the device. Refer to [Command Service](../builtin/command-service) for details on how to configure this service in Kura.
```
Comment (marcellorinaldo) [inline docs/administration/remote-management-kapua.md]: Broken link, use `../core-services/command-service.md`

**Review by marcellorinaldo — CHANGES_REQUESTED** [review]

**Review by marcellorinaldo — APPROVED** [review]

**Pull request merged**